passkeeper/git.py

```diff
@@ -27,6 +27,11 @@ def add(self, files):
         for file in files:
             self._run_git_cmd('add %s' % file)
 
+    def clean(self):
+        self.init()
+        self._run_git_cmd('add -A')
+        self.commit('Clean git History') 
```

Comment:
I prefer let git.py generic and simple. A clean or a add need to do just clean or add. If you want a commit, no magic function, do a commit after in your code.